numba_dppy/tests/test_rename_numpy_function_pass.py

```diff
@@ -75,6 +76,7 @@ def got(a):
         self.assertTrue(check_equivalent(expected_ir, pipeline.state.func_ir))
 
 
+@unittest.skipUnless(ensure_dpnp(), 'test only when dpNP is available')
```

Comment:
We need to test the rewrites regardless of whether dpnp is available. This is so because we enable this functions through dpnp and we need to rewrite them in order to fail if dpnp is available.